thirdeye/thirdeye-frontend/app/mocks/filterBarConfig.js

```diff
@@ -12,6 +13,7 @@ export default [
   {
     header: "Anomalies",
     eventType: "anomaly",
+    color: "teal",
```

Comment:
hard coded for now?

Reply:
Yes. In the future, when the config is more finalized, this is assumed to be in the config.